src/WebJobs.Script.WebHost/Middleware/FunctionInvocationMiddleware.cs

```diff
@@ -61,7 +61,7 @@ public async Task Invoke(HttpContext context)
                     return;
                 }
 
-                ActionContext actionContext = new ActionContext(context, _emptyRouteData, _emptyActionDescriptor);
+                ActionContext actionContext = new ActionContext(context, context.GetRouteData(), new ActionDescriptor());
```

Comment:
Remove the (now) unused variables `_emptyRouteData` and `_emptyActionDescriptor`, though for the descriptor, we might want to just continue using the previously created/allocated one.